lib/storage/src/content_manager/raft_state.rs

```diff
@@ -69,7 +69,10 @@ impl Persistent {
         &self.state.0
     }
 
-    pub fn load_or_init(storage_path: impl AsRef<Path>) -> Result<Self, StorageError> {
+    pub fn load_or_init(
+        storage_path: impl AsRef<Path>,
+        first_peer: bool,
```

Comment:
Can you please add a rust doc for the `load_or_init` function explaining the exact semantic of the `first_peer` argument :pray: